com.brave.Brave.yaml

```diff
@@ -20,15 +20,11 @@ finish-args:
   - '--talk-name=org.freedesktop.Notifications'
   - '--talk-name=org.freedesktop.secrets'
   - '--talk-name=org.freedesktop.UPower'
-  - '--talk-name=org.freedesktop.GeoClue2'
   - '--talk-name=org.gnome.SessionManager'
   - '--system-talk-name=org.freedesktop.Avahi'
-  - '--own-name=org.mpris.MediaPlayer2.*'
-  - '--filesystem=host-etc'
-  - '--filesystem=xdg-documents'
-  - '--filesystem=xdg-download'
-  - '--filesystem=xdg-music'
-  - '--filesystem=xdg-videos'
+  - '--system-own-name=org.mpris.MediaPlayer2.chromium.*'
+  - '--system-own-name=org.freedesktop.GeoClue2'
+  - '--filesystem=host'
```

Comment:
Doesn't that make the sandboxing somewhat nonexistent? You're giving the Flatpak rw access to the whole filesystem (ro if directories/files are owned by someone else)

Reply:
This true but the problem is that I use chromium to load PDFs from other places like `/media` and sometimes `/mnt`. 

So I figured that I might as well give full access because you never know what a user might want to load.

Reply:
I've decided to only allow /etc and xdg-download. The user decides what they want to override later, not my job

Reply:
> This true but the problem is that I use chromium to load PDFs from other places like /media and sometimes /mnt.

Can you try

```json
- '--filesystem=/media:ro'
- '--filesystem=/mnt:ro`
```
and check if it works?

Reply:
It would work but not for everybody. Let's be restrictive by default and only allow rw to ~/Downloads and then let the user allow whatever they want with Flatseal  

That's what Firefox is doing